src/tooltip/tooltip.js

```diff
@@ -302,6 +299,18 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
             attrs.$observe( prefix+'Title', function ( val ) {
               ttScope.title = val;
+              if (prefix === 'popover') {
+                $timeout(positionTooltip, 0, false);
```

Comment:
Does [$applyAsync](https://docs.angularjs.org/api/ng/type/$rootScope.Scope#$applyAsync) work instead of $timeout ?